fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchRevIncludeTest.java

```diff
@@ -891,6 +894,7 @@ public void testSearchRevIncludeIteratePrimaryType() {
         WebTarget target = getWebTarget();
         Response response =
                 target.path("Patient")
+                .queryParam("_total", "none")
```

Comment:
same comment as above - could add an assertNull(bundle.getTotal()) in this testcase

Reply:
Added.